src/garage/experiment/local_tf_runner.py

```diff
@@ -193,24 +199,25 @@ def save(self, epoch, paths=None):
         """
         assert self.has_setup
 
-        logger.log('Saving snapshot...')
+        if isinstance(self.snapshot_config, SnapshotConfig):
+            logger.log('Saving snapshot...')
```

Comment:
I suggest adding a warning of non-resumable training when `SnapshotConfig` is missing.

Reply:
The case is not possible if we use `experiment_wrapper` to run experiment. Now the `SnapshotConfig` will be passed to the Runner with default option even users don't want to snapshot. I just realised this is a problem.

Reply:
After some discussion with KR, I made some changes and added a warning in `__init__`. Please see if it resolves your concern.